e2e-tests/src/test/resources/streamx-mesh.yml

```diff
@@ -0,0 +1,21 @@
+defaultRegistry: ghcr.io/streamx-dev/streamx
+defaultImageTag: latest-jvm
```

Comment:
This version should be fixed to `streamx.version` from parent pom.
To inject this version here:
1. Add to your e2e pom a <build> tag: 
 ```
    <testResources>
      <testResource>
        <directory>src/main/resources</directory>
        <excludes>
          <exclude>filtered/*</exclude>
        </excludes>
      </testResource>
      <testResource>
        <filtering>true</filtering>
        <directory>src/main/resources/filtered</directory>
        <includes>
          <include>streamx-mesh.yml</include>
        </includes>
      </testResource>
    </testResources>
```
2. Move this file to `src/test/resources/filtered`
3. Replace `defaultImageTag: latest-jvm` with `defaultImageTag: ${streamx.version}-jvm`